elastic/integrations
# Adding first draft of [OTEL Kubernetes] Cluster Overview Dashboard

- Enhancement

## Proposed commit message

Adding a new `[OTEL Kubernetes] Cluster Overview` dashboard in k8s integration

Please explain:

- WHAT: A new dashboard that visualises the needed metrics for K8s Cluster observability
- WHY:  To support the user experience of Kubernetes Observability users that use OTEL collector 

## Checklist



## How to test this PR locally

1. Create a kind cluster `kind create cluster`
2. Clone this repo
3. Navigate to `elastic/integrations/packages/kubernetes`
4. Build package by running `elastic-package build`
5. Install the elastic-agent-otel distribution from https://github.com/elastic/k8scollector/blob/otel/helm/elastic-agent/elastic-otel-collector_all.yaml. Run `kubectl apply -f elastic-otel-collector_all.yaml`
6. Spin up a new `elastic-package stack up -d -v --version=8.15.0-SNAPSHOT`
7. The new Kubernetes 1.64.0 integration should be available. Install it
8. Navigate to Dashboards and open `[OTEL Kubernetes] Cluster Overview `


## Screenshots

![Screenshot 2024-07-10 at 1 35 47 PM](https://github.com/elastic/integrations/assets/1685415/5775e8fd-00e8-4551-93e4-e7ee9821ae77)
